frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -1,22 +1,34 @@
 import React from "react";
-import { Sighting } from "../../clients/apiClient";
+import { Sighting, SightingFromExternalApi } from "../../clients/apiClient";
 import "./BrowseSightings.scss";
 
 interface SightingProps {
-  sighting: Sighting;
+  sighting: Sighting | SightingFromExternalApi;
 }
 
 export const SightingCard: React.FunctionComponent<SightingProps> = ({
   sighting,
 }) => {
+  const imageUrl: string | undefined =
+    (sighting as Sighting).imageUrl ??
+    (sighting as SightingFromExternalApi).photoUrl;
+
+  const description: string | undefined =
+    (sighting as Sighting).description ??
+    (sighting as SightingFromExternalApi).location.description;
```

Comment:
This location description doesn't replace the description of the actual sighting, so should be used in another way